File: src/main/java/org/sonar/plugins/xquery/language/XQuerySourceImporter.java
/*
 * © 2014 by Intellectual Reserve, Inc. All rights reserved.
 */

package org.sonar.plugins.xquery.language;

import org.sonar.api.batch.AbstractSourceImporter;

public class XQuerySourceImporter extends AbstractSourceImporter {

    /**
     * Instantiates a new xquery source importer.
     */
    public XQuerySourceImporter(XQuery xQuery) {
        super(xQuery);
    }

    /**
     * {@inheritDoc}
     */
    @Override
    public String toString() {
        return "XQuery Source Importer";
    }
}

File: src/main/java/org/sonar/plugins/xquery/language/XQuerySourceImporter.kt
/*
 * © 2014 by Intellectual Reserve, Inc. All rights reserved.
 */

package org.sonar.plugins.xquery.language

import org.sonar.api.batch.AbstractSourceImporter

class XQuerySourceImporter(xQuery: XQuery) : AbstractSourceImporter(xQuery) {

    /**
     * {@inheritDoc}
     */
    override fun toString(): String {
        return "XQuery Source Importer"
    }
}